backend/src/main/java/de/sambam/gamenighttracker/controller/UserController.java

```diff
@@ -53,13 +53,12 @@ public UserController(UserService userService) {
         }
     }
 
-    @PostMapping("/game/{apiGameId}/gamesessions")
-    public Optional<GameSession> addNewGameSession(@RequestBody GameSession newSession,
-                                                   @PathVariable String apiGameId,
+    @PostMapping("/savesession")
```

Comment:
Maybe prefer a resource oriented route style, a POST to `/session` creates a new session? Creating is in line with POST semantics,  no need to use a verb in the route, imho.

Reply:
I already have one endpoint called "/sessions"... so I wanted to be more distinctive... could the same endpoint be targeted by both types of requests GET & POST?